include/block.h

```diff
@@ -373,7 +373,7 @@ class Block: public App
 		 *
 		 * @return masterState & (SERVERD_STATUS_MASK | SERVERD_ONOFF_MASK)
 		 */
-		const rts2_status_t getMasterState ()
+		rts2_status_t getMasterState ()
```

Comment:
I still don't understand the need to remove const. Patch otherwise looks great and I will be more than happy to accept it, but can you please explain the reason for const removal?